ntbs-service/DataAccess/ReferenceDataRepository.cs

```diff
@@ -162,6 +177,20 @@ public async Task<IList<PHEC>> GetAllPhecs()
                 .ToListAsync();
         }
 
+        public async Task<IList<PHEC>> GetPhecsBySearchKeywords(List<string> searchKeywords)
+        {
+            IQueryable<PHEC> queryable = _context.PHEC;
+            
+            foreach (var keyword in searchKeywords)
+            {
+                queryable = queryable.Where(h => h.Name.ToLower().Contains(keyword));
```

Comment:
After our discussion earlier, I checked and we are using SQLite in our integration tests, so the issue here is actually more complicated that we originally thought.

In fact, the `Contains` method is translated in SQLite to the use of `instr`, which is case-sensitive (even though the equivalent function in a MySQL database is case-insensitive). The solution here is to tell EF to use `LIKE` instead (which was what I was expecting it to do). We can do this by using e.g.
```
queryable = queryable.Where(h => EF.Functions.Like(h.Name, $"%{keyword}%"));
```

I think we should make this change, as it really isn't good to have production code that's only there to make tests pass.